src/scripts/main.js

```diff
@@ -1 +1,88 @@
 'use strict';
+
+let bodyDoc;
+let rightClick;
+let leftClick;
```

Comment:
The variables 'rightClick' and 'leftClick' are declared but never reset to 'false'. This can lead to incorrect behavior if 'thirdPromise' is called multiple times.